README.md

```diff
@@ -1,2 +1,33 @@
-# peer-server
+# Peer-server
+
 Node server for Peer
+
+## Scripts
+
+> yarn run build
```

Comment:
these are all nits, but the `run` part of the command is optional for all of these.
```suggestion
> yarn build
```